packages/js-sdk/test/InstantDistributionAgreementV1Helper.test.js

```diff
@@ -312,23 +312,23 @@ contract("InstantDistributionAgreementV1Helper helper class", (accounts) => {
             const subscriber = bob;
             await sf.ida.createIndex({
                 superToken: superToken.address,
+                publisher,
                 indexId,
-                sender: publisher,
             });
 
             await sf.ida.updateSubscription({
                 superToken: superToken.address,
+                publisher,
                 indexId,
                 subscriber,
                 units: units.toString(),
-                sender: publisher,
             });
 
             await sf.ida.approveSubscription({
                 superToken: superToken.address,
                 indexId,
                 publisher,
-                sender: subscriber,
+                subscriber: subscriber,
```

Comment:
if the sender is always the subscriber, then this is ok

Reply:
yes it is.

The purpose of these changes is to keep all the parameters consistent. Only some operation would require "sender" parameter to tell from which account the transaction should be (including claim and deleteSubscription)